api/services/doc_analysis.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import nltk
+import fitz
+from io import BytesIO
+
+from api.utils.file_util import read_text_file
+from nlp.resources.constants import KEYWORDS_PATH
+
+LOGGER = logging.getLogger(__name__)
+
+
+def load_keywords():
+    keywords = set(read_text_file(KEYWORDS_PATH).split('\n'))
+    # TODO: more sophisticated keyword matching 
+    return keywords
+
+
+KEYWORDS = load_keywords()
+
+
+def highlight_keywords(input_pdf_path, output_pdf_path):
+	pdfDoc = fitz.open(input_pdf_path)
+	for pg in range(pdfDoc.page_count):
```

Comment:
aici pare identare cu tab iar in rest avem identare cu 4 spatii;
sugestie: foloseste scriptul `./scripts/format.sh` care ar trebui sa auto formateze tot